### ISSUE TITLE
display error: the pane content exceed the prompt cursor

### ISSUE THREAD

**Issue by BostonLobster** [tmux/tmux#1901]
### Issue description

Please read https://github.com/tmux/tmux/blob/master/.github/CONTRIBUTING.md
before opening an issue.

If you have upgraded, make sure your issue is not covered in the CHANGES file
for your version: https://raw.githubusercontent.com/tmux/tmux/master/CHANGES

Describe the problem and the steps to reproduce. Add a minimal tmux config if
necessary. Screenshots can be helpful, but no more than one or two.

Do not report bugs (crashes, incorrect behaviour) without reproducing on a tmux
built from the latest code in Git.

### Required information

Please provide the following information:

* tmux version (`tmux -V`). tmux 2.6
* Platform (`uname -sp`). Linux x86_64
* $TERM inside and outside of tmux (`echo $TERM`). inside: screen, outside: xterm
* Logs from tmux (`tmux kill-server; tmux -vv new`).

I use tmux on my server and connect to my server using MobaXterm. Recently, I am always facing this problem: I minimize the MobaXterm window and leave for a while, when I come back, I find the tmux pane content exceed the prompt line, as shown in the below image

![tmux](https://user-images.githubusercontent.com/47711465/64471776-2cddc680-d188-11e9-90bc-f75c91ba7594.png)

The orange cursor indicates the prompt line, and you can find there are some pane contents under this promptl ine which is very annoying.

**Comment by nicm** [comment]
Does this still happen with 3.0-rc or master?

**Comment by BostonLobster** [comment]
> Does this still happen with 3.0-rc or master?

@nicm I didn't try 3.0-rc or master because I don't know how to compile from source. I have tried downgrading Tmux to 2.1 which is the version I usually use and I have several other remote servers work fine with this version.

I finally find this display error is caused by the `win + D` or `win + M` shortcuts in Windows 10 together with MobaXterm. When I use `win + D` or `win + M` to minimize all the opened windows, and then I click the mobaxterm icon to restore its window, the display error in tmux panel happens. But only one remote server in MobaXterm has this error (regardless of Tmux 2.6 or 2.1 or 2.3 or 2.7...) My other remote servers with Tmux 2.1 in mobaxterm are not affected by `win + D`...

So, my compromised solution is to exit tmux session before pressing `win + D` to minimize all windows when operating on that server

If you have any suggestion after reviewing my situation, I would be very appreciate!

**Comment by BostonLobster** [comment]
Actually in my other remote server, If I use `win + D` to minimize mobaxterm and then restore it, the cursers in the active tmux pane will have a little bit displacement (moved left by a few column), but clicking that pane and scrolling my mouse will correct its position. 

But the wierd thing in that problematic server is, the curser will moved down by several lines.... Causing the pane content exceed the curser line

**Comment by nicm** [comment]
2.6 is 2 years old and 2.1 is 4 years old, I am not interested in bug
reports from either of them. Please try a newer tmux.

**Comment by BostonLobster** [comment]
@nicm I can understand your concerns. Actually, in my problematic remote server, I have tested Tmux 2.1, 2.3, 2.6, and 2.7... All of them will affected by the `win + D` shortcut in MobaXterm on Windows 10. The OS in that problematic server is Ubuntu 18.04 with default Tmux 2.6, I installed higher version of Tmux by adding source from [here](https://packages.ubuntu.com/search?keywords=tmux&searchon=names).

I may follow your suggestion by installing more higher version of Tmux.
